### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by tkirshboim** [BreezeWhite/oemer#11]
I'd like to try out oemer but I'm having an issue installing it.
Here's the command that I'm running and the error message that I'm getting (MacOS Monterey 12.4 / 2,8 GHz Quad-Core Intel Core i7):
```
$ pip install oemer
Collecting oemer
  Using cached oemer-0.1.3-py3-none-any.whl (1.3 MB)
Collecting onnxruntime
  Using cached onnxruntime-1.12.0-cp310-cp310-macosx_10_15_universal2.whl (6.0 MB)
Collecting scikit-learn==0.24.2
  Using cached scikit-learn-0.24.2.tar.gz (7.5 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting oemer
  Using cached oemer-0.1.2-py3-none-any.whl (1.3 MB)
  Using cached oemer-0.1.1-py3-none-any.whl (1.3 MB)
  Using cached oemer-0.1.0-py3-none-any.whl (1.3 MB)
ERROR: Cannot install oemer==0.1.0, oemer==0.1.1, oemer==0.1.2 and oemer==0.1.3 because these package versions have conflicting dependencies.

The conflict is caused by:
    oemer 0.1.3 depends on scipy==1.6.2
    oemer 0.1.2 depends on scipy==1.6.2
    oemer 0.1.1 depends on scipy==1.6.2
    oemer 0.1.0 depends on onnxruntime-gpu

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

Thanks in advance for the help! 🙂

**Comment by Jaaba101** [comment]
Hi, 
I'm getting the same error on Win10. I've installed the Python 3.10 from MS Store.

**Comment by ohuang1224** [comment]
It only seems to work on Python 3.8, uninstall 3.10 and install 3.8 from the MS store.

**Comment by Jaaba101** [comment]
@ohuang1224 thanks, got it installed with 3.8!

**Comment by va5ja** [comment]
Did anyone manage to get oemer working on Apple silicon? I have Python@3.8 on my M1 and I can't install oemer. Would it be possible to use ranges for dependencies (numpy, scipy) or maybe upgrade deps to newer versions?

**Comment by Feles-Mala** [comment]
Hi, I had similar installation issues, but looking at the scipy==1.6.2 requirements shows it needs python < 3.10, so I downgraded to 3.9.13 and it works. (windows 10)

**Comment by BreezeWhite** [comment]
The installation issue should be fixed in the new `v0.1.5` release. See comment [here](https://github.com/BreezeWhite/oemer/issues/16#issuecomment-1356736288).
